scripts/trivia.js

```diff
@@ -737,6 +737,16 @@ TriviaGame.prototype.startTriviaRound = function () {
                 questionNumber = Trivia.randomId();
                 i++;
             }
+            // to account for webclient until it gets updated to Gen 7
+            if ((triviaq.get(questionNumber) === null || (triviaq.get(questionNumber).category.toLowerCase() === "who's that pokémon?" && this.webclientPlayers())) && i !== 200) {
+                var qu = triviaq.get(questionNumber);
+                if (qu.question > 721) { // the stored questions for who's that pokemon? are the dex numbers. Basically, "if this is > Gen 7"
+                    var ran = sys.rand(1, 721);
+                    // all the who's that pokemon questions are stored sequentially because they were created with /createwtp
+                    // so let's change to a pre-Gen 7 pokemon
+                    questionNumber = questionNumber - ran;
```

Comment:
Er, how exactly does this work? Let's say the Pokemon is Torracat and ran is 1, won't that change the question to Litten?